example-config.yaml

```diff
@@ -77,6 +93,17 @@ stormpath:
             placeholder: "Last Name"
             required: true
             type: "text"
+          # Unless specifically set to false, the organizationNameKey field will
+          # automatically be shown when stormpath.web.multiTenancy.enabled=true,
+          # and an organization cannot be resolved (the default organization
+          # resolver does not return an organization).
+          organizationNameKey:
+            enabled: null
```

Comment:
If we don't add it to the field list, the developer has no way to supress (or order) this field.  This is because I'm proposing that we automatically show this field when a organization cannot be resolved.  I'm open to discussing this point.